client/features/auth/SignIn.jsx

```diff
@@ -27,16 +29,27 @@ class SignIn extends React.Component {
   };
 
   handleSubmit() {
-    axios.post('/login', this.state)
+    axios.post('/auth/login', this.state)
     .then((res) => {
       console.log('/login Res', res);
+      if (!res.data) {
```

Comment:
The only way *anything* will be returned as data is if the password is correct?

Reply:
Yes, the data is a boolean. true is pw matches.